content/blog/eng/automation-testing-a-step-back.md

```diff
@@ -0,0 +1,114 @@
+---
+authors: ["carmine-laface"]
+date: "2020-06-23"
+draft: false
+share: true
+categories: [English, Software Testing]
+title: "Automation testing: a step back"
+type: "post"
+languageCode: "en-EN"
+toc: true
+---
+
+The goal of this article is to define basic concepts related to testing, trying not to take anything for granted.
+
+# Why we test?
+
+Why is it important to write automated tests? I asked myself and I did some research because the answer to this question was not obvious to me. I knew it was important but I didn't know why. So I decided to try to motivate it starting from a point of view as impartial as possible.
+
+After some research I came to the conclusion that the benefits of the tests are four: they can avoid regression, they can count as documentation, they can help write better code and they can help find bugs during development.
```

Comment:
After some research, I came to the conclusion that the benefits of tests are four